Sources/TSCBasic/Path.swift

```diff
@@ -534,36 +513,17 @@ private struct WindowsPath: Path, Sendable {
         return String(cString: representation)
     }
 
-    init(normalizingAbsolutePath path: String) {
-        self.init(string: Self.repr(path).withCString(encodedAs: UTF16.self) { pwszPath in
-          var canonical: PWSTR!
-          _ = PathAllocCanonicalize(pwszPath,
-                                    ULONG(PATHCCH_ALLOW_LONG_PATHS.rawValue),
-                                    &canonical)
-          return String(decodingCString: canonical, as: UTF16.self)
-        })
-    }
-
     init(validatingAbsolutePath path: String) throws {
-        let realpath = Self.repr(path)
+        let fsr: UnsafePointer<Int8> = path.fileSystemRepresentation
+        defer { fsr.deallocate() }
+
+        let realpath = String(cString: fsr)
```

Comment:
Please revert this part of the change.  fsr on an empty string is not permitted, this can now incorrectly invoke the fsr.

Reply:
did you just add this a few days ago? I dont have an opinion about the WindowsPath imp, so if this need stop be changed please let me know how